extensions/index.js

```diff
@@ -10,7 +10,7 @@ fs.readdirSync(__dirname).forEach(moduleDir => {
         try {
             indexStat = fs.statSync(
                 path.join(__dirname, moduleDir, 'index.js'));
-        } catch (err) {} // eslint-disable-line no-empty
+        } catch {} // eslint-disable-line no-empty
```

Comment:
There is definitely a better way to implement this function and avoid empty catch:

```js
fs.readdirSync(__dirname).forEach(moduleDir => {
    const extPath = path.join(__dirname, moduleDir);
    const indexPath = path.join(extPath, 'index.js');
    
    if (fs.statSync(extPath).isDirectory() && 
        fs.existsSync(indexPath)) {
        extensions[moduleDir] = require(`./${moduleDir}`);
    }
});
```